### PR TITLE
job #9947 updated xtuml2masl man page

### PR DESCRIPTION
**Pull request by leviathan747**

### CODE REVIEW

**Comment by leviathan747** [comment]
https://github.com/xtuml/mc/pull/282

File: src/org.xtuml.bp.doc/Reference/MASL/xtuml2masl/xtuml2masl.md [hunk 1]
```diff
@@ -9,18 +9,21 @@ Syntax
...
 ### Domain Export
 ```
-xtuml2masl [-v | -V] [-e] -i <eclipse project> -d <domain component> [ -o <output directory> ]
+xtuml2masl [-v | -V] [-e] [-xf] [-xl] [-xt] -i <eclipse project> -d <domain component> [ -o <output directory> ]
 ```
    
 ### Package Export
 ```
-xtuml2masl [-v | -V] [-e] -i <eclipse project> -p <project package> [ -o <output directory> ] 
+xtuml2masl [-v | -V] [-e] [-xf] [-xl] [-xt] -i <eclipse project> -d <project package> [ -o <output directory> ]
```
Comment (keithbrown) [inline src/org.xtuml.bp.doc/Reference/MASL/xtuml2masl/xtuml2masl.md]: why did you change the project export from -p to -d?  Looks to me like -p is still valid (used on lin 1213)

Comment (leviathan747) [reply src/org.xtuml.bp.doc/Reference/MASL/xtuml2masl/xtuml2masl.md]: copy and paste error
